resources/Archives.py

```diff
@@ -24,7 +41,15 @@ def get(self):
             return "There has been an error pulling data!"
 
     @api_required
-    def put(self):
+    def put(self) -> Dict[str, str]:
+        """
+        Updates the archive data based on the provided JSON payload.
+
+        Expects a JSON payload with archive item identifiers and updates them in the database.
```

Comment:
can you replace "item" with "data source"?

Reply:
Done 🐧